scripts/gen_c_api.py

```diff
@@ -64,6 +64,12 @@
         "field_ext.h",
         "vec_ops_ext.h",
         "ntt_ext.h",
+    },
+    "m31": {
+        "ntt_ext.h",
+        "ntt.h",
```

Comment:
should there be ntt for m31 or not? based on here yes but the m31.h and also rust is missing ntt. Why is that?

Reply:
m31 doesn't support the current ntt algorithm, fft over the extension field is planned

Reply:
@nonam3e  - both of them radix2 and mixed-radix ntt?

> m31 doesn't support the current ntt algorithm, fft over the extension field is planned

Reply:
@vhnatyk it's not possible to generate omega, that is why it's not possible to support both of them